requirements.txt

```diff
@@ -0,0 +1,2 @@
+fondant[docker]==0.6.2
```

Comment:
can we remove the `docker` extras, Its only needed for publishing image which is not needed here